tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -184,6 +185,33 @@ def test_hit_scaling(scheduler, region, instance, pcluster_config_reader, cluste
         num_dynamic_nodes=3,
         dynamic_instance_type=instance,
     )
+    assert_no_errors_in_logs(remote_command_executor, scheduler)
+
+
+@pytest.mark.regions(["us-west-1"])
+@pytest.mark.instances(["c5.xlarge"])
+@pytest.mark.schedulers(["slurm"])
+@pytest.mark.oss(["alinux2"])
+@pytest.mark.usefixtures("region", "os")
+@pytest.mark.hit_rare_failure
+def test_hit_rare_failure(scheduler, region, instance, pcluster_config_reader, clusters_factory, test_datadir):
+    """Test that slurm-specific scaling logic can handle rare failures."""
+    cluster_config = pcluster_config_reader(scaledown_idletime=3)
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    scheduler_commands = get_scheduler_commands(scheduler, remote_command_executor)
+
+    _assert_cluster_initial_conditions(scheduler_commands, instance, 10, 10, 1, 1)
+    _test_status_check_replacement(
```

Comment:
this test probably is not using the static node. Is the node used by the next one or we can remove it?

Reply:
Static instance is used in `_test_clustermgtd_down_logic`